### PR TITLE
DOP-4647: Deleting unused Landing components

### PR DESCRIPTION
**Pull request by mayaraman19**
### Ticket

DOP-4647

### Notes

Yay deletion!

### README updates

- - [ ] This PR introduces changes that should be reflected in the README.md and/or HACKING.md, and I have made those updates.
- - [X] This PR does not introduce changes that should be reflected in the README.md and/or HACKING.md

### CODE REVIEW

**Review by i80and — APPROVED** [review]
:shipit: 🎉

**Comment by mayaraman19** [comment]
waiting on design confirmation before merging!

**Pull request merged**
